src/fleet/azext_fleet/_help.py

```diff
@@ -61,6 +61,12 @@
       short-summary: Kubernetes configuration file to update. Use "-" to print YAML to stdout instead.
 """
 
+helps['fleet wait'] = """
+type: command
+short-summary: Wait for a fleet resouce to reach a desired state.
```

Comment:
is that short summary standard from other CLI?

what is the desired state?
should that be "terminal state" (Succeeded, Failed, or Canceled)?
or can the user provide arguments?

Reply:
yes, short-summary is similar to other services like AKS.

yes, desired state is a terminal state. user can provide arguments like --created. you can find more info here `az fleet wait -h`